instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c

```diff
@@ -1921,13 +2156,30 @@ static void fillResponsePayload(response_payload* res_payload, ngx_http_request_
         res_payload->otel_attributes = ngx_pcalloc(r->pool, ((conf->nginxModuleAttributes->nelts + 1)/3) * sizeof(http_headers));
         ngx_uint_t otel_attributes_idx=0;
 
-        resolve_attributes_variables(r);
         for (ngx_uint_t j = 0, isKey = 1, isValue = 0; j < conf->nginxModuleAttributes->nelts; j++) {
+
+            ngx_str_t var_name = (((ngx_str_t *)(conf->nginxModuleAttributes->elts))[j]);
+            ngx_uint_t           key; // The variable's hashed key.
+            ngx_http_variable_value_t  *value; // Pointer to the value object.
+
+            if(var_name.data[0] == '$'){
```

Comment:
what is the meaning of dollar?  Is it delimiter? Please add some varieable and use it

Reply:
'$' is used to identify Nginx Variables, as they have to start woth '$' sign. Will a comment work here ?